src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -3018,6 +3018,18 @@ namespace SurfaceGeometry {
             }
         }
     }
+
+    Real64 rotAzmDiff(Real64 const &AzmA, Real64 const &AzmB)
```

Comment:
Please add a comment to indicate that expected units in and out. It's obvious from the test values that it's in degrees, but there are other places in the code that work in radians (or apply conversions back and forth). Or perhaps the function name should include "Deg" or "Degrees"?

Also, can you apply this new function in the weather manager?

Reply:
Added comment lines to indicate the input and output angle units for the function; also added a `Deg` suffix to the function name to indicate this (Degree). 

The weather manager has an interpolation function that already uses this concept. There does not seem to be a need to revise the code there.